src/windows_aligned_file_reader.cpp

```diff
@@ -6,6 +6,8 @@
 #include "windows_aligned_file_reader.h"
 #include <iostream>
 #include "utils.h"
+#include <locale>
```

Comment:
Is this included because of the use of `wcstomb_s`? (Go ahead and mark resolved if so)

Reply:
remove unuse header.